index.html

```diff
@@ -20,7 +20,15 @@
                 <li><a href="#Portfolio">Portfolio</a></li>
                 <li><a href="#About-myself">About</a></li>
                 <li><a href="#Contact">Contact</a></li>
-              </ul>  
+              </ul> 
+            </div>
+            <div class="menu-popup" id="menu-popup">
+              <span class="close">&times;</span>
+              <ul class="mobile_specific-links">
+                <li class="nav-link"><a href="#Portfolio">Portfolio</a></li>
+                <li class="nav-link"><a href="#about">About</a></li>
+                <li class="nav-link"><a href="#Contact">Contact</a></li>
+              </ul>
```

Comment:
- [ ] You've done a fantastic job, as acknowledged by the previous reviewer. However, it is imperative that the options on your mobile menu direct the user to the correct section of the page. This implies that the portfolio option should lead to the works section, the about option to the about me section, and the contact option to the form :+1:.